Automattic/Gravatar-SDK-iOS
# [Tooling] Setting up Danger

### Description

PR setting up [Danger](https://danger.systems) and [Dangermattic](https://github.com/Automattic/dangermattic) checks in the repo.
See pdnsEh-1TU-p2.

### Testing Steps

- Make sure there's a Danger status check posted in the GitHub build
- Make sure that, if applicable, a message from the bot Dangermattic bot is posted in the PR
- Make sure changes in the PR (e.g. removing, adding labels) trigger Danger to run again
